extensions/omni_credentials/CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.0] - 2025-3-17
```

Comment:
Please follow the format as described and insert changes into 0.2.0 as that's the target release (you can see it is TBD)